operator/pkg/apis/operator/v1alpha1/defaults.go

```diff
@@ -227,3 +229,20 @@ func setDefaultsKarmadaDescheduler(obj *KarmadaComponents) {
 		descheduler.Replicas = pointer.Int32(1)
 	}
 }
+
+func setDefaultsKarmadaMetricsAdapter(obj *KarmadaComponents) {
+	if obj.KarmadaMetricsAdapter == nil {
+		obj.KarmadaMetricsAdapter = &KarmadaMetricsAdapter{}
+	}
+
+	metricsAdapter := obj.KarmadaMetricsAdapter
+	if len(metricsAdapter.Image.ImageRepository) == 0 {
+		metricsAdapter.Image.ImageRepository = KarmadaMetricsAdapterImageRepository
+	}
+	if len(metricsAdapter.Image.ImageTag) == 0 {
+		metricsAdapter.Image.ImageTag = constants.KarmadaDefaultVersion
+	}
+	if metricsAdapter.Replicas == nil {
+		metricsAdapter.Replicas = pointer.Int32(1)
```

Comment:
Let's set it 2 replicas. Can u help update `artifacts/deploy/karmada-metrics-adapter.yaml` also?

Reply:
That‘s sure.